ui/Modal/Org/Create.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="typescript">
+  import type { Identity } from "../../src/identity";
+
+  import { Copyable, Modal } from "../../DesignSystem/Component";
+  import { Avatar, Button, Emoji } from "../../DesignSystem/Primitive";
+
+  import * as org from "../../src/org";
+  import { ellipsed } from "../../src/style";
+  import * as modal from "../../src/modal";
```

Comment:
These should all use absolute imports.

Reply:
b50aed685be4b0b179941e05d82e4b8425cf4e28